labeller/labeller.go

```diff
@@ -331,6 +339,114 @@ func (l *Labeller) sync(img *image.Image) (*secscan.Layer, error) {
 	return layerData, nil
 }
 
+// MirroredImages checks if there is a mirror configuration for the provided image. Returns a list
+// of configured mirrors or an empty slice if no mirror was found.
+func (l *Labeller) MirroredImages(img *image.Image, mirrors map[string][]string) []*image.Image {
```

Comment:
This function handles only rules like `example.com` and `example.com/foo`, but it can be nested arbitrarily deep.

Reply:
solved, ptal

Reply:
If I have

```
spec:
  repositoryDigestMirrors:
    - mirrors:
      - mirror.example.com/foo-bar-baz
      source: registry.example.com/foo/bar/baz
```

will it map `registry.example.com/foo/bar/baz/qux/xyzzy` to `mirror.example.com/foo-bar-baz/qux/xyzzy`?

Reply:
I don't know what is the status of nested repository support. I am closing this one until I get an answer to that.